app/src/main/res/values/strings.xml

```diff
@@ -10,196 +10,196 @@
-    <string name="collector_info">UDP Collector: %1$s:%2$d</string>
-    <string name="http_server_status">HTTP Server: http://%1$s:%2$d</string>
+    <string name="collector_info">UDP collector: %1$s:%2$d</string>
+    <string name="http_server_status">HTTP server: http://%1$s:%2$d</string>
     <string name="rcvd_and_sent">%1$s received — %2$s sent</string>
     <string name="query">Query</string>
     <string name="host">Host</string>
     <string name="capture_not_running">Capture not running!</string>
-    <string name="capture_not_started">Capture not started!</string>
+    <string name="capture_not_started">Start capturing network data first.</string>
     <string name="app_filter">Target app</string>
     <string name="ready">Ready</string>
     <string name="about">About</string>
     <string name="status">Status</string>
     <string name="connections_view">Connections</string>
     <string name="no_connections">No connections</string>
-    <string name="connection_details">Connection Details</string>
+    <string name="connection_details">Connection details</string>
     <string name="app">App</string>
     <string name="protocol">Protocol</string>
     <string name="source">Source</string>
     <string name="destination">Destination</string>
     <string name="bytes">Bytes</string>
     <string name="packets">Packets</string>
     <string name="duration">Duration</string>
-    <string name="http_server">HTTP Server</string>
-    <string name="udp_exporter">UDP Exporter</string>
+    <string name="http_server">HTTP server</string>
+    <string name="udp_exporter">UDP exporter collector</string>
     <string name="no_dump">None</string>
     <string name="no_dump_info">PCAP will not be dumped</string>
     <string name="http_server_info">Start an HTTP server for the PCAP download</string>
     <string name="udp_exporter_info">Sends the PCAP to a remote UDP receiver</string>
-    <string name="http_server_port">HTTP Server Port</string>
-    <string name="receiver_ip_address">Collector IP Address</string>
-    <string name="receiver_port">Collector Port</string>
-    <string name="dump_mode">Dump Mode</string>
-    <string name="user_guide">User Guide</string>
-    <string name="open_telegram_group">Telegram Group</string>
-    <string name="source_code">Source Code</string>
-    <string name="yes">YES</string>
-    <string name="no">NO</string>
-    <string name="existing_vpn_confirm">The currently active VPN app will be disconnected. Do you want to proceed?</string>
-    <string name="vpn_setup_failed">VPN setup failed</string>
-    <string name="app_not_found">App "%1$s" not found</string>
-    <string name="apps_loading_please_wait">Apps loading in progress, please wait</string>
+    <string name="http_server_port">Port</string>
+    <string name="receiver_ip_address">IP address</string>
+    <string name="receiver_port">Port</string>
+    <string name="dump_mode">Dumping mode</string>
```

Comment:
To "dump data" is rather unfortunate, and may introduce errors in translation.

Reply:
Please explain why. "Dump traffic" is a common term in network related software to indicate "save a copy of the traffic"

Reply:
It also has the meaning of (and in some cases translates to) ~discard all the data.